src/styles/app/10-settings-controls.css

```
@@ -35,19 +35,19 @@
 
 .earlyAccessBadge {
 	--early-access-color: var(--interactive-accent);
-	padding: 3px 8px;
-	border-radius: var(--radius-sm);
-	border: 1px solid
-		color-mix(in srgb, var(--early-access-color) 52%, transparent);
+	padding: 4px 8px;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)